qemu/tests/driver_in_use.py

```
@@ -0,0 +1,143 @@
+                return True
+        except Exception, err:
+            raise exceptions.TestFail("Check process error: %s" % str(err))
+        return False
+
+    def bg_stress_test():
+        """
+        Run backgroud test.
+        """
+        bg_stress_test = params.get("run_bgstress")
+        if bg_stress_test:
+            try:
+                error_context.context("Run test %s background" % bg_stress_test,
+                                      logging.info)
+                stress_thread = ""
+                wait_time = float(params.get("wait_bg_time", 60))
+                target = params.get("target", "")
+                bg_stress_run_flag = params.get("bg_stress_run_flag")
+                env[bg_stress_run_flag] = False
+                stress_thread = utils.InterruptedThread(
+                    utils_test.run_virt_sub_test, (test, params, env),
+                    {"sub_type": bg_stress_test})
+                stress_thread.start()
+                if stress_thread.is_alive():
+                    try:
+                        stop_time = time.time() + wait_time
+                        while time.time() < stop_time:
+                            status = check_bg_running(session, target)
+                            if status:
+                                env["bg_status"] = 1
+                                break
+                    except Exception:
+                        raise exceptions.TestFail("Backgroud test %s is not"
+                                                  "alive!" % bg_stress_test)
+                if not utils_misc.wait_for(lambda: env.get(bg_stress_run_flag),
+                                           wait_time, 0, 5,
+                                           "Wait %s test start" % bg_stress_test):
+                    err = "Fail to start %s test" % bg_stress_test
+                    raise exceptions.TestError(err)
+                return stress_thread
+            except:
+                pass
```

why you ignore all exception here ?